python/paddle/pir/math_op_patch.py

```diff
@@ -334,6 +334,9 @@ def _scalar_div_(var, value):
     def _scalar_neg_(var):
         return paddle.scale(var, -1.0, 0.0)
 
+    def _scalar_abs_(var):
```

Comment:
```suggestion
    def _abs_(var):
```

感觉不需要用 `scalar` 前缀

其他的我也不知道为啥用 scalar 前缀，可能是 scale OP 第 2、3 个位置被转换为了 `Scalar`？但至少 abs OP 是和 scalar 没啥关系的

无关紧要，可以下个 PR 改一下就好